src/services/pool/decorators/pool.decorator.ts

```diff
@@ -70,15 +70,8 @@ export class PoolDecorator {
       poolService.setVolumeSnapshot(poolSnapshot);
       await poolService.setLinearPools();
 
-      if (setAprCondition) {
-        await poolService.setAPR(
-          poolSnapshot,
-          prices,
-          currency,
-          protocolFeePercentage,
-          gaugeBALAprs[pool.id],
-          gaugeRewardTokenAprs[pool.id]
-        );
+      if (setAprCondition && poolSnapshot) {
```

Comment:
`protocolFeePercentage, gaugeBALAprs, gaugeRewardTokenAprs` were used in calculation on frontend, it seems there is no more need in them so we can remove `getData` function and `setAprCondition`. 

also I am not sure why `poolSnapshot` is added in if statement?

Reply:
here should be `if (includeAprs)`